scripts/cluster-up-packet.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Copyright (c) 2018 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ ! -z "${CIRCLE_BUILD_NUM}" ] ; then
+    CLOUD_DEPLOYMENT_NAME=nsm-ci-${CIRCLE_BUILD_NUM}
+else
+    CLOUD_DEPLOYMENT_NAME=nsm-$$
+fi
+
+CLOUD_DEPLOYMENT_DATA=$(pwd)/.data
+
+docker run \
+  -v ${CLOUD_DEPLOYMENT_DATA}:/cncf/data \
+  --dns 147.75.69.23 --dns 8.8.8.8 \
+  -e NAME=${CLOUD_DEPLOYMENT_NAME} \
+  -e CLOUD=packet    \
+  -e COMMAND=deploy \
+  -e BACKEND=file  \
+  -e PACKET_AUTH_TOKEN=${PACKET_AUTH_TOKEN} \
```

Comment:
Maybe add a note that PACKET_AUTH_TOKEN and PACKET_PROJECT_ID are defined in the circle-ci dashboard? May be worth a broader note that these would need to be defined before this could be run by hand.

Reply:
Good point will add it